src/main/java/fr/foodlimit/api/place/PlaceController.java

```diff
@@ -44,17 +50,14 @@ public PlaceController(TokenProvider tokenProvider) {
    * @return
    */
   @GetMapping("/{id}")
-  public Place getPlace(@PathVariable("id") Long id) {
-    return placeService.getPlace(id);
-  }
+  public ResponseEntity<Place> getPlace(HttpServletRequest request, @PathVariable("id") Long id) {
+    Place place = placeService.getPlace(id);
 
-  /**
-   * Supprime un foyer de l'utilisateur
-   * @param id
-   */
-  @DeleteMapping("/{id}")
-  public void deletePlace(@PathVariable("id") Long id) {
-    placeService.deletePlace(id);
+    if (!this.environment.getActiveProfiles()[0].equals("test")) {
+      if (!checkPlace(request, place)) return ResponseEntity.status(HttpStatus.FORBIDDEN).build();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)